cmd/daemon/import.go

```diff
@@ -0,0 +1,177 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/gofrs/flock"
+	"github.com/pactus-project/pactus/cmd"
+	"github.com/pactus-project/pactus/genesis"
+	"github.com/pactus-project/pactus/util"
+	"github.com/spf13/cobra"
+)
+
+func buildImportCmd(parentCmd *cobra.Command) {
+	importCmd := &cobra.Command{
+		Use:   "import",
+		Short: "download and import pruned data",
+	}
+	parentCmd.AddCommand(importCmd)
+
+	workingDirOpt := addWorkingDirOption(importCmd)
+	serverAddrOpt := importCmd.Flags().String("server-addr", "", "custom import server")
```

Comment:
It should have a default value.
Update the help string: `import server address`